libbeat/scripts/Makefile

```diff
@@ -43,6 +43,9 @@ DOCKER_COMPOSE?=docker-compose -f ${PWD}/../testing/environments/base.yml -f ${P
 DOCKER_CACHE?=1 # If set to 0, all docker images are created without cache
 GOPACKAGES_COMMA_SEP=$(subst $(space),$(comma),$(strip ${GOPACKAGES}))
 PYTHON_ENV?=${BUILD_DIR}/python-env
+BUILDID?=$(shell git rev-parse HEAD)
+CROSSCOMPILE_TARGETS?="linux/amd64 linux/386 windows/amd64 windows/386 darwin/amd64"
+CROSSCOMPILE_TARGETS_CGO?="linux/amd64 linux/386"
```

Comment:
Agree. The name of these variables is not good. I need to find a better name. I think I left somewhere a TODO for it.